### PR TITLE
Goverance

### PR DESCRIPTION
**Pull request by djaglowski**
## Description of Changes
Adds governance, updates maintainers, and polishes up documentation.

## **Please check that the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Add a changelog entry (for non-trivial bug fixes / features)
- [x] CI passes

### CODE REVIEW

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.7241725|+0.017254233|126.65908|-3.0130692|
|1|5000|6.2071285|+0.10356951|133.0629|-4.6087036|
|1|10000|12.672936|+0.17265892|144.599|-0.9366913|
|1|50000|59.77714|-8.103886|176.1778|-7.300659|
|1|100000|116.7266|-1.9376907|230.60439|-6.085541|
|10|100|2.3103862|-0.06896329|132.62971|+0.37190247|
|10|500|6.9483914|+0.017212868|137.32921|+0.7027283|
|10|1000|14.207259|+0.7242012|147.80711|+0.6639252|
|10|5000|66.13947|+2.9658928|185.63672|+7.0051117|
|10|10000|129.21648|+3.1777878|230.342|+17.65126|

File: MAINTAINERS.md [hunk 1]
```diff
@@ -7,4 +7,5 @@ This file lists the maintainers of the Stanza project. The responsibilities of m
 | ---- | --------- | ----------- |
 | [Mike Kelly](mailto:mike.kelly@bluemedora.com) | [mkelly](https://github.com/mkelly) | observIQ |
 | [Dan Jaglowski](mailto:dan.jaglowski@bluemedora.com) | [djaglowski](https://github.com/djaglowski) | observIQ |
-| [Camden Cheek](mailto:camden.cheek@bluemedora.com) | [camdencheek](https://github.com/camdencheek) | observIQ |
\ No newline at end of file
+| [Camden Cheek](mailto:camden.cheek@bluemedora.com) | [camdencheek](https://github.com/camdencheek) | observIQ |
```
Comment (camdencheek) [inline MAINTAINERS.md:10]: Should these use our `@observiq.com` emails?

**Review by camdencheek — APPROVED** [review]

**Review by mkelly — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (djaglowski) [reply MAINTAINERS.md:10]: I'm going to leave these for now. I think each maintainer should update their own as they like.

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.7414098|+0.08620751|125.20124|-5.060753|
|1|5000|6.155263|+0.155159|136.68845|-0.122161865|
|1|10000|12.569168|+0.3101864|141.72427|-1.7660217|
|1|50000|61.55457|+3.3136826|176.94801|+2.7889404|
|1|100000|127.537605|+12.471176|241.67458|+12.963104|
|10|100|2.413832|+0.103404045|134.89587|+0.94503784|
|10|500|6.9139214|-0.00018262863|138.2178|-0.04499817|
|10|1000|13.741704|+0.20691586|144.7632|-0.47817993|
|10|5000|65.933815|+0.10396576|185.20016|+0.009292603|
|10|10000|127.38286|+1.6671753|231.59833|+3.4618835|

**Pull request merged**